common/lib/controller/encrypt.controller.js

```diff
@@ -30,38 +30,29 @@ define(function(require, exports, module) {
     sub.SubController.call(this, port);
     this.editorControl = null;
     this.recipientsCallback = null;
+    // register event handlers
+    this.on('eframe-recipients', this.displayRecipientProposal);
+    this.on('eframe-display-editor', this.openEditor);
```

Comment:
@toberndo I wanted to get your feedback on the new event handling style before using it broadly. Events can now be registered like so. I stole the syntax from angular.js. This is much more concise and easier to test than ...